src/mrnet/network/reaction_network.py

```diff
@@ -1541,6 +1541,44 @@ def find_paths(self, starts, target, weight, num_paths=10, ignorenode=[]):  # ->
 
         return self.PRs, paths, top_path_list
 
+    def path_analysis(self):
```

Comment:
Maybe give the users some ability to control how the ranking is done? For instance, maybe people won't care how long the paths are as long as they're thermodynamically viable.